express/experiments/ccx0027/blocks/quick-action1/quick-action.css

```diff
@@ -0,0 +1,271 @@
+/* ccl-quick-action styles */
+
+.quick-action ccl-quick-action {
+    display: block;
+    margin: 0 auto;
+    width: 712px;
+    height: 10px;
+    position: relative;
+}
+
+ccl-quick-action .button-container {
+    align-items: baseline;
+    width: 535px;
+}
+
+ccl-quick-action .button-container a[data-action='Download'], ccl-quick-action .quick-action-complete-overlay a[data-action='Download']{
```

Comment:
Do we need the intermediary `.button-container` and `.quick-action-complete-overlay`? otherwise we can combine and optimize the selectors

Reply:
we show actions in places, one is button-container comes on the top of the utility and the other one is in the overlay, which will be part of the complete-overlay